lib/route/src/Obelisk/Route/Frontend.hs

```diff
@@ -216,14 +217,36 @@ subRoute_ :: (MonadFix m, MonadHold t m, GEq r, Adjustable t m) => (forall a. r
 subRoute_ f = factorRouted $ strictDynWidget_ $ \(c :=> r') -> do
   runRoutedT (f c) r'
 
--- | Like 'subRoute_', but with a pair rather than an R
-subPairRoute_ :: (MonadFix m, MonadHold t m, Eq a, Adjustable t m) => (a -> RoutedT t b m ()) -> RoutedT t (a, b) m ()
-subPairRoute_ f = withRoutedT (fmap (\(a, b) -> Const2 a :/ b)) $ subRoute_ (\(Const2 a) -> f a)
-
```

Comment:
Just moved this below, so the `subPairRoute*` are together (and the deprecation notice is easier to follow).